content/raw/chat/online_users.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Online Users spec"
+version: "1.0.0"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Online Users
+
+## Motivation
+
+It's important for end users to know which community members are online or offline, they can use this to judge how active a community is, if a member is likely following a discussion and whenever to message that member or not. At the same time introducing such a feature might have privacy concerns which we need to be aware of, so this feature is implemented as a opt-in functionality (see notes).
+
+## Designs
+
+* [member section](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=6644%3A46846)
```

Comment:
Both of these figma links are out of date and should be removed from this spec.  The latest figma's are here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=106%3A20383 and here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=670%3A334981